instance/models/mixins/openedx_config.py

```diff
@@ -235,7 +235,7 @@ def _get_configuration_variables(self):
             # e.g if the instance receives an average of around 20000 requests per day,
             # with 10% for static assets, restarting after 10000 requests means
             # restarting each of the 3 LMS workers about every .8 days.
-            "EDXAPP_LMS_MAX_REQ": 5000,
+            "EDXAPP_LMS_MAX_REQ": 1000,
```

Comment:
This is to account for the increased number of workers?
Seems like a good idea

Reply:
@kaizoku yes.